src/main/java/de/unistuttgart/overworldbackend/data/CourseDTO.java

```diff
@@ -21,13 +22,24 @@ public class CourseDTO {
   @NotNull
   String courseName;
 
-  String description;
+  @Pattern(regexp = "^(WS|SS)-[0-9][0-9]$")
```

Comment:
```suggestion
  @Pattern(regexp = "^(WS|SS)-[0-9][0-9]+$")
```
Given that this regex is used in multiple places, we should probably make it a constant...